lib/src/dartdoc_options.dart

```diff
@@ -598,16 +598,17 @@ abstract class DartdocOption<T> {
       _OptionValueWithContext<Object> valueWithContext, String missingFilename);
 
   /// Call [_onMissing] for every path that does not exist.
-  void _validatePaths(_OptionValueWithContext<dynamic> valueWithContext) {
+  void _validatePaths(_OptionValueWithContext<Object> valueWithContext) {
     if (!mustExist) return;
     assert(isDir || isFile);
     List<String> resolvedPaths;
-    if (valueWithContext.value is String) {
+    var value = valueWithContext.value;
+    if (value is String) {
       resolvedPaths = [valueWithContext.resolvedValue];
-    } else if (valueWithContext.value is List<String>) {
-      resolvedPaths = valueWithContext.resolvedValue.toList();
-    } else if (valueWithContext.value is Map<String, String>) {
-      resolvedPaths = valueWithContext.resolvedValue.values.toList();
+    } else if (value is List<String>) {
+      resolvedPaths = (valueWithContext.resolvedValue as List).toList();
```

Comment:
toList() seems like it is redundant here now?

Reply:
Done.